### ISSUE TITLE
Screen shot in Community Installation

### ISSUE THREAD

**Issue by chtelmko** [open-learning-exchange/open-learning-exchange.github.io#2971]
**Problem:**
There are more bullet points than there are red arrows in the screenshot .

**Steps to reproduce the problem:**
Going to [Community Installation, under Tech Geniuses](https://open-learning-exchange.github.io/#!pages/techgenius/tg-install.md)

**Screenshots:**

![screenshot missing arrows](https://user-images.githubusercontent.com/40531136/80122671-dcbb4500-855b-11ea-9381-eea0d12d0cf5.png)

**proposed solution:**

Update screenshot or combine some of the bullet points into one : 

Select Nations = MadagascarBell, Language = Your local language, Region = Your region of the world;

**Comment by gautamsethi1** [comment]
I think this is a good catch as the arrows are a bit misleading. I think adding more arrows is the right way to go to fix this issue. Good Job!

**Comment by lmmrssa** [comment]
screenshot itself is old and is not valid

**Comment by vinitnprabhu** [comment]
Good catch! An updated screenshot will make it easier for a vi to complete that step

**Comment by 3011089503** [comment]
The screenshot makes instructions more easy to understand!
